mlfinlab/tests/test_fracdiff.py

```diff
@@ -0,0 +1,92 @@
+"""
```

Comment:
I am happy with this if you feel that the unit tests here cover enough edge cases.